saskatoon/sitebase/locale/fr/LC_MESSAGES/django.po

```diff
@@ -618,10 +614,9 @@ msgstr "Points d'Equipement (Annexe B)"
 
 #: sitebase/templates/app/index/pickleader.html:8
 msgid "Planning Harvests Using the Properties List (Annex D)"
```

Comment:
capitalization probably not necessary in the original:  "Planning harvests using the Properties List (Annex D)"